weaver/core/drivers/fabric-driver/server/helpers/fabric-functions.ts

```diff
@@ -0,0 +1,714 @@
+/*
```

Comment:
This and few other files that are copied from fabric-cli, these were good for you to get easily implement functions in driver, but when we want to merge it to main, can you filter out what is needed and what is not and add those only? And see if similar function is already available in other TS files in fabric-driver (because mostly it would), and use those if possible.

Its **not urgent**, it can be done at the very last stage also when PR is ready to be merged.